schema/revision.ttl

```diff
@@ -0,0 +1,123 @@
+@prefix : <https://rdf.equinor.com/ontology/revision/> .
+@prefix owl: <http://www.w3.org/2002/07/owl#> .
+@prefix rdf: <http://www.w3.org/1999/02/22-rdf-syntax-ns#> .
+@prefix xml: <http://www.w3.org/XML/1998/namespace> .
+@prefix xsd: <http://www.w3.org/2001/XMLSchema#> .
+@prefix prov: <http://www.w3.org/ns/prov> .
+@prefix rdfs: <http://www.w3.org/2000/01/rdf-schema#> .
+@prefix skos: <http://www.w3.org/2004/02/skos/core#> .
+@prefix prov: <http://www.w3.org/ns/prov#> .
+@base <https://rdf.equinor.com/ontology/revision> .
+
+<https://rdf.equinor.com/ontology/revision> rdf:type owl:Ontology ;
+                                             owl:versionIRI <https://rdf.equinor.com/ontology/revision/2023-01-10> ;
+                                             owl:imports <https://rdf.equinor.com/ontology/record-syntax/2022-12-05> .
+
+#################################################################
+#    Annotation properties
+#################################################################
+
+###  http://www.w3.org/2004/02/skos/core#prefLabel
+skos:prefLabel rdf:type owl:AnnotationProperty .
+
+
+#################################################################
+#    Datatypes
+#################################################################
+
+###  http://www.w3.org/2001/XMLSchema#date
+xsd:date rdf:type rdfs:Datatype .
```

Comment:
Do we need to type xsd - types? Remove?

Reply:
This is generated from protege, but lets try to remove it.